.github/workflows/docs.yml

```diff
@@ -15,7 +15,9 @@ jobs:
       with:
         fetch-depth: 0
     - name: Document which reference
-      run: echo "Running for ${{ github.ref_name }}"
+      uses: rlespinasse/github-slug-action@v4
+      run: echo "Running for ${{ github.ref_name }}
+           Uses URL Reference ${{ env.GITHUB_REF_SLUG_URL }}"
```

Comment:
I'm seeing an error in the [workflow](https://github.com/TIDES-transit/TIDES/actions/runs/3153704737) that a step can't have both `uses` and `run` keys